operator/bundle/manifests/platform.stackrox.io_centrals.yaml

```
@@ -686,10 +686,14 @@ spec:
                   type: object
                 type: array
               misc:
-                description: Miscellaneous settings.
+                description: Deprecated field. It is no longer necessary to specify
+                  it. This field will be removed in a future release. Miscellaneous
+                  settings.
```

It reads a bit weird. How about:
```suggestion
                description: Deprecated field. This field will be removed in a future release. Miscellaneous
                  settings.
```

---

The manifests are generated. I've updated the generation source to rephrase the message in https://github.com/stackrox/stackrox/pull/8766/commits/b10734963574a9ca248cb38bc0bedfcad52522d4